src/utils/permissions.ts

```diff
@@ -1,101 +0,0 @@
-// todo https://github.com/AtB-AS/kundevendt/issues/8615:
-
-export enum PermissionType {
-  Bluetooth = 'bluetooth',
-  LocationWhenInUse = 'locationWhenInUse',
-  LocationAlwaysAllow = 'locationAlwaysAllow',
-  MotionAndFitnessActivity = 'motionAndFitnessActivity',
-}
-
-async function placeholderReturnQuickly(val: boolean): Promise<boolean> {
-  return new Promise((resolve) => {
-    setTimeout(() => {
-      resolve(val);
-    }, 1);
-  });
-}
-
-async function placeholderReturnAfter1Sec(val: boolean): Promise<boolean> {
-  return new Promise((resolve) => {
-    setTimeout(() => {
-      resolve(val);
-    }, 1000);
-  });
-}
-
```

Comment:
Ideally we should have implemented these functions instead of removing them, but ok 😅